h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -257,6 +262,18 @@ void doScoreOrCopyMetrics(Job job) {
     }
   }
 
+  private DistributionFamily familyToDistribution(GLMModel.GLMParameters.Family aFamily) {
+    if (aFamily == GLMModel.GLMParameters.Family.binomial) {
+      return DistributionFamily.bernoulli;
+    }
+    try {
+      return Enum.valueOf(DistributionFamily.class, aFamily.toString());
+    }
+    catch (IllegalArgumentException e) {
+      throw new H2OIllegalArgumentException("Don't know how to find the right DistributionFamily for Family: " + aFamily);
+    }
+  }
+
   private DistributionFamily distributionFamily(Model aModel) {
     // TODO: hack alert: In DRF, _parms._distribution is always set to multinomial.  Yay.
     if (aModel instanceof DRFModel)
```

Comment:
wondering if this logic is not outdated: DRF seems to only exclude `quasi-binomial` and custom distributions currently.
As this is used to identify the SE distribution, this can have a bad impact. wdyt?

Reply:
I am not sure. I know there are checks for quasi binomial and custom distribution but in documentation there is no mention of DRF supporting distribution (no mention on DRF's page[1]) and on distribution's page[2] only GBM, DeepLearning and XGBoost are mentioned.

[1] https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/drf.html?highlight=distribution#
[2] https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/distribution.html?highlight=distribution

Reply:
documentation is more likely to be late than ahead.
@maurever? apparently you're the last one who added validation for DRF distributions, do you know what it supports currently?